### PR TITLE
Fix urdf include

### PR DESCRIPTION
**Pull request by sjahr**
### Description

Based on #3003 

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extend the tutorials / documentation [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Document API changes relevant to the user in the [MIGRATION.md](https://github.com/moveit/moveit2/blob/main/MIGRATION.md) notes
- [ ] Create tests, which fail without this PR [reference](https://moveit.picknik.ai/humble/doc/examples/tests/tests_tutorial.html)
- [ ] Include a screenshot if changing a GUI
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/moveit/moveit2/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

File: moveit2.repos [hunk 1]
```diff
@@ -12,3 +12,7 @@ repositories:
     type: git
     url: https://github.com/moveit/moveit_resources.git
     version: ros2
+  srdfdom:
+    type: git
+    https://github.com/moveit/srdfdom.git
```
Comment (sjahr) [inline moveit2.repos:17]:
```suggestion
    url: https://github.com/moveit/srdfdom.git
```

**Comment by rhaschke** [comment]
Now, it's only srdfdom, which still seems to use the deprecated headers.

**Review by rhaschke — COMMENTED** [review]
Do we need the selective include here in the moveit2 repo?
As far as I understood, that was only necessary for srdfdom, which only has a single branch for ros2. Here, we have different branches for rolling and older releases.

**Comment by sjahr** [comment]
Closed in favor of #3003

**Pull request closed without merging**